rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
+- Start Date: 09 February 2018
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Begin using an RFC process. Initially use a structure similar to the process used by the maintainers of IPFS lang, as described in https://github.com/IPFS-lang/rfcs
+
+# Motivation
+[motivation]: #motivation
+
+Establish a clear, repeatable process for people to propose ideas and make decisions in a way that
+1. Allows everyone to see what decisions have been made and how they were made
+2. Allows everyone to see what proposals have been proposed and comment on them or ssuggest changes
+3. Works for a globally distributed group of contributors to discuss and make decisions asynchronously
+4. Can be replicated/repeated in sibling projects or spinoff projects like libp2p, IPLD, etc.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+_[Cribbed directly from [IPFS-lang/rfcs README](https://github.com/IPFS-lang/rfcs#IPFS-rfcs)]_
```

Comment:
Same note as up on line 9 — is this supposed to be `ipfs/rfcs/README.md` or `rust-lang/rfcs/README.md`?